Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Generator.cs

```diff
@@ -93,9 +93,9 @@ public void Execute(GeneratorExecutionContext context)
                 var isExecutable = false;
 
                 bool foundFatalError = false;
-                
```

Comment:
theres a bunch of whitespace issues in this PR which i am still working on fixing - this is also affecting the unit tests which is why they are failing. I need to figure out the whitespace issues so the tests pass and then i can write new tests